test/00-feed.js

```diff
@@ -16,6 +16,13 @@ tape('00 feed type', function (t) {
 
   t.deepEquals(bfe.decode(encoded), values, 'decode works')
 
+  /* unhappy paths */
+  t.throws(() => bfe.encode('@dogFeed'), 'unknown feedId encode throws')
```

Comment:
Great with these tests, I'm thinking we might need to tighten the check that leads to the feeds branch (similar with the message etc.) because it is rather unfortunate that we can't have strings that starts with @ without being a feed.

Reply:
We can use is-canonical-base64 style code. Recently did this with box test

Prefix @
Suffix \\.\\w+

(it shows strings which get converted into regexp

Reply:
Pushed up a commit with tests and a solution to some of that

Reply:
https://github.com/ssb-ngi-pointer/ssb-bfe/pull/18/commits/08c19a531ebf19ab48153d0ee887eda2c5cfc215

Reply:
Great thanks! The functions in util are similar to the ones in [ssb-ref](https://github.com/ssb-js/ssb-ref/blob/33e800db6025d3c77d27383c16063d3b39f4b06f/index.js#L8) but not quite the same because of classical. We should standardize on those once https://github.com/ssb-ngi-pointer/ssb-meta-feed-spec/issues/19 is settled.